package.json

```diff
@@ -9,6 +9,7 @@
     "dist"
   ],
   "scripts": {
+    "postinstall": "rm -f node_modules/seaport/foundry.toml",
```

Comment:
This is a non-ideal workaround for resolving some conflicting behaviour that was occurring due to the foundry.toml nested in the seaport package